java/arcs/core/entity/EntityBase.kt

```diff
@@ -30,14 +30,14 @@ open class EntityBase(
     private val schema: Schema
 ) : Entity {
     // Private var _entityId with public getter. Only this class can set this field.
-    private var _entityId: String? = null
+    protected var _entityId: String? = null
```

Comment:
Comment doesn't match up.

But also, the reason I did this in the first place was to prevent developers being able to change the entity ID. Is this change necessary?

Reply:
You caught me... I don't like this either, but I was trying to minimize the amount of codegen editing I had to do ;-)

Fixed this up to keep the setters private.